apps/server/lib/google-drive/src/internal/utils/log.ts

```
@@ -0,0 +1,124 @@
+	retryOfRequestLogID?: string | undefined;
+	retryOf?: string | undefined;
+	url?: string | undefined;
+	status?: number | undefined;
+	method?: string | undefined;
+	durationMs?: number | undefined;
+	message?: unknown;
+	body?: unknown;
+}) => {
+	if (details.options) {
+		details.options = { ...details.options };
+		delete details.options["headers"]; // redundant + leaks internals
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)